sql/core/src/test/resources/sql-tests/inputs/show-views.sql

```diff
@@ -0,0 +1,28 @@
+-- Test data.
+CREATE DATABASE showdb;
+USE showdb;
+CREATE TABLE tbl(a String, b Int, c String, d String) USING parquet;
+CREATE VIEW view_1 AS SELECT * FROM tbl;
+CREATE VIEW view_2 AS SELECT * FROM tbl WHERE c='a';
+CREATE TEMPORARY VIEW view_3(e int) USING parquet;
+CREATE GLOBAL TEMP VIEW view_4 AS SELECT 1 as col1;
+
+-- SHOW VIEWS
+SHOW VIEWS;
+SHOW VIEWS FROM showdb;
+SHOW VIEWS IN showdb;
+SHOW VIEWS IN global_temp;
+
+-- SHOW VIEWS WITH wildcard match
+SHOW VIEWS 'view_*';
+SHOW VIEWS LIKE 'view_1*|view_2*';
+SHOW VIEWS IN showdb 'view_*';
```

Comment:
Can you add tests for `SHOW VIEWS` having both `IN` and (optional) `LIKE`?

Reply:
Sure, updated/added tests for `show-views.sql` and `show-tables.sql` in c5570d249426a821896ab85e37bf2e0c1d9a39ee.